core/spec/models/spree/shipment_spec.rb

```diff
@@ -1,5 +1,5 @@
-require 'spec_helper'
-require 'benchmark'
+require "spec_helper"
```

Comment:
I'm all for cleaning up the specs but I'd prefer we didn't make sweeping updates to stuff like quotation marks.  We don't have a rule about this (and I don't think we need one) and it makes commits harder to review and git blames more tedious.  I'm all for updating stuff like that when you're already touching a line of code but I don't think it's super helpful to just make changes like this by themselves.